lib/tasks/rake_helper.rb

```diff
@@ -188,7 +188,7 @@ def image_name
   end
 
   def user
-    @user ||= Etc.getpwnam(Etc.getlogin)
+    @user ||= Etc.getpwnam(Etc.getpwuid.name)
```

Comment:
Both `getpwnam` and `getpwuid` return a `Passwd` object I believe so you might be able to simplify with this:
```suggestion
    @user ||= Etc.getpwuid
```